sonic_installer/main.py

```diff
@@ -480,7 +480,9 @@ def sonic_installer():
 @sonic_installer.command('install')
 @click.option('-y', '--yes', is_flag=True, callback=abort_if_false,
               expose_value=False, prompt='New image will be installed, continue?')
-@click.option('-f', '--force', is_flag=True,
+@click.option('--force_secureboot', is_flag=True,
```

Comment:
>force_secureboot [](http://example.com/codeflow?start=17&length=16)

After double thinking, we should keep old short option and long option, so backward-compatiable. We can modify the help text.